pkg/templating/template_description.go

```diff
@@ -126,6 +127,14 @@ func EnrichedIssueWithFirstFound(firstFound time.Time) enrichedIssueOption {
 	}
 }
 
+// EnrichedIssueWithFixedCodeAnnotation allows to define which of the annotations contains fixed code
+func EnrichedIssueWithFixedCodeAnnotation(fixedCodeAnnotation string) enrichedIssueOption {
+	return func(ei *enrichedIssue) error {
+		ei.FixedCodeAnnotation = fixedCodeAnnotation
```

Comment:
Should we check for the string not to be empty?

Reply:
this annotation could easily be empty however, not every workflow needs to have a component that generates fixed code :-)